l10n_it_fatturapa_sale/views/sale_order_views.xml

```diff
@@ -34,9 +34,7 @@
                     string="Electronic Invoice"
                     groups="account.group_account_user"
                 >
-                    <group name="related_documents" string="Related Documents">
-                        <field name="related_documents" nolabel="1" colspan="2" />
-                    </group>
+                    <field name="related_documents" nolabel="1" />
```

Comment:
Come mai qui la stringa "Related Documents" è stata tolta e non è stata messa in `separator` come (ad es.) [qui](https://github.com/OCA/l10n-italy/pull/3699/files#diff-0d6e3045af0140c933caf9b651726880e7fcc975d491c8db37e0bde0bf486646R443)?

Reply:
> posso inserirlo ma visivamente mi sembrava più idoneo alla form

In effetti è più carino così e non toglie informazioni visto che la stringa è presente anche nel nome della scheda.

Anzi, sarebbe carino farlo anche per la 14.0, se ti avanza tempo. :wink: 

Vedi:
![immagine](https://github.com/OCA/l10n-italy/assets/227940/333cdbb7-4dc8-423d-b700-64e4539bb413)
![immagine](https://github.com/OCA/l10n-italy/assets/227940/15a97692-2e92-4645-b67d-f8acbec8e38a)